Armadillo/src/main/java/com/scribd/armadillo/di/AppModule.kt

```diff
@@ -33,7 +33,7 @@ internal class AppModule(private val context: Context) {
     @Singleton
     @PrivateModule
     @Provides
-    fun stateStore(reducer: Reducer): ArmadilloStateStore = ArmadilloStateStore(reducer)
+    fun stateStore(reducer: Reducer): ArmadilloStateStore = ArmadilloStateStore(reducer, context)
```

Comment:
Actually, you could even pass the looper from here, since that's what you actually need the context for

Reply:
That reads better yea